### PR TITLE
Update Software_Carbon_Intensity_Specification.md

### PR DESCRIPTION
**Pull request by srini1978**
Added reference to the document around how and where we can use the SCI open data project to get standard reference numbers for emissions factors, energy values, embodied emissions etc

### CODE REVIEW

**Comment by Henry-WattTime** [comment]
Condense paragraph and move to end of line 25. Change 'will provide' to 'provides'

**Review by srini1978 — COMMENTED** [review]
The feedback has been incorporated I believe. If not please let me know if I am missing something

**Comment by srini1978** [comment]
> @srini1978 I believe the changes requested [here](https://github.com/Green-Software-Foundation/software_carbon_intensity/pull/265#issuecomment-1191699612) are yet to be made?

I might have missed the changes. Can someone repost here so that I can make the changes?

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ The SCI encourages calculation using granular real-world data, which is challeng
 
 Suppose, there is a lack of access, capability, or rights to the necessary real-world data. In that case, the SCI allows for data generated through modeling using best estimates instead.
 
+There are standard reference numbers that would be needed through the process of calculation of SCI score for example emissions per GB transferred , manufacturing emissions per GB of RAM. Wherever possible, these numbers can be obtained from SCI open data project https://github.com/Green-Software-Foundation/sci-data . If reference numbers are not available, then  a request for data can be made.
```
Comment (Henry-WattTime) [inline Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:29]:
```suggestion
There is data that may be needed through the process of calculation of SCI score. For example, people calculating a score may need emissions per GB transferred or manufacturing emissions per GB of RAM. These numbers can be obtained from SCI open data project https://github.com/Green-Software-Foundation/sci-data. If reference numbers are not available, then a request for data can be made.
```

**Pull request merged**